docs/section9/Rule6-jl.md

```diff
@@ -2,38 +2,51 @@
+- **Applicability Check 1:** Check if any Hotel/Model Guestroom or Dormitory Living Quarters, or Dwelling Units exist in the building segment in the Proposed model: ```for building_segment_proposed in P_RMR.building.building_segments:```  
 
-  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
+- For each thermal_block in building segment in the Proposed model: ```thermal_block_proposed in building_segment_proposed.thermal_blocks:```
 
-  - Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
+- For each zone in thermal block: ```zone_proposed in thermal_block_proposed.zones:```
 
-  - Get space from thermal zone: ```space in thermal_zone.spaces:```  
+- For each space in zone: ```space_proposed in zone_proposed.spaces:```  
 
-    - Get the space type of the space: ```space_type = space.space_type```  
+  - Check if lighting_space_type of the space is applicable: ```if ( space_proposed.lighting_space_type == "Guest Room" or "Dormitory Living Quarters" or "Dwelling Unit" ): lighting_power_allowance_proposed = space_proposed.lighting_space_type```  
 
-    - If the space_type is "Guest Room", or "Dormitory Living Quarters", or "Dwelling Unit": ```if space_type == "Guest Room" or space_type == "Dormitory Living Quarters" or space_type == "Dwelling Unit"```  
+    - If the lighting_space_type is "Guest Room", or "Dormitory Living Quarters": ```if ( lighting_space_type_proposed == "Guest Room" or "Dormitory Living Quarters" ):```  
 
-      - Get the lighting power denstiy allowance: ```lighting_power_allowance = data_lookup(table_9_6_1, space_type)```
+      - Get the lighting power denstiy allowance for the proposed model: ```lighting_power_allowance_proposed = data_lookup(table_9_6_1, lighting_space_type_proposed)```  
 
-      - Get the total lighting power modeled in the space: ```space_lighting_power_proposed = space.lighting_power```  
+    - If the lighting_space_type is "Dwelling Unit": ```else: lighting_power_allowance_proposed = 0.6```  
 
-      - Get the area of the space: ```floor_area_proposed = space.floor_area```  
+    - Get interior_lighting in space: ```interior_lighting_proposed = space_proposed.interior_lightings```  
 
-      **Rule Assertion:** For each space: ```space_lighting_power_proposed <= floor_area_proposed * lighting_power_allowance```  
+      - Get the total design power_per_area for the space: ```space_lighting_power_per_area_proposed = sum( lighting.power_per_area for lighting in interior_lighting_proposed )```  
+
+      **Rule Assertion:** For each space that is Hotel/Motel Guestroom or Dormitory Living Quarters, or Dwelling Unites in the proposed model: ```space_lighting_power_per_area_proposed <= lighting_power_allowance_proposed```  
+
```

Comment:
@xingcx and @JasonGlazer - should this be space_lighting_power_per_area_proposed >= lighting_power_allowance_proposed (greater than and not less than?)
App G says "lighting power used in the simulation shall be equal to the lighting power allowance in Table 9.6.1 for the appropriate space type or as designed, whichever is greater". so at the very minimum, the LPD would be equal to the the prescriptive requirement in Table 9.6.1 or greater than that value.

Reply:
I agree it should be >=

@xingcx please make that change.

Reply:
@JasonGlazer and @supriyagoel , shouldn't it be the maximum of the lighting power in U-RMR and lighting power allowance in Table 9.6.1 for that space type?

Reply:
@mkarpman that was our intention for the proposed change to use ">="

Reply:
Updated the proposed value to the max of baseline allowance or user model lighting power.